docs/sources/get-started/quick-start.md

```diff
@@ -2,160 +2,277 @@
+   To learn more about the query editor, refer to the [query editor documentation](https://grafana.com/docs/grafana/latest/datasources/loki/query-editor/).
 
 1. The Loki query editor has two modes (3):
 
    - [Builder mode](https://grafana.com/docs/grafana/latest/datasources/loki/query-editor/#builder-mode), which provides a visual query designer.
    - [Code mode](https://grafana.com/docs/grafana/latest/datasources/loki/query-editor/#code-mode), which provides a feature-rich editor for writing LogQL queries.
 
-   Next we’ll walk through a few simple queries using both the builder and code views.
-
 1. Click **Code** (3) to work in Code mode in the query editor.
 
-    Here are some basic sample queries to get you started using LogQL.  Note that these queries assume that you followed the instructions to create a directory called `evaluate-loki`. If you installed in a different directory, you’ll need to modify these queries to match your installation directory.  After copying any of these queries into the query editor, click **Run Query** (4) to execute the query.
+   Here are some sample queries to get you started using LogQL.
+   These queries assume that you followed the instructions to create a directory called `evaluate-loki`.
 
-    1. View all the log lines which have the container label "flog":
+   If you installed in a different directory, you’ll need to modify these queries to match your installation directory.
 
-        ```bash
-        {container="evaluate-loki-flog-1"}
-        ```
+   After copying any of these queries into the query editor, click **Run Query** (4) to execute the query.
 
-        In Loki, this is called a log stream. Loki uses [labels](https://grafana.com/docs/loki/<LOKI_VERSION>/get-started/labels/) as metadata to describe log streams.  Loki queries always start with a label selector.  In the query above, the label selector is `container`.
+   1. View all the log lines which have the container label `evaluate-loki-flog-1`:
 
-    1. To view all the log lines which have the container label "grafana":
+      <!-- Killercoda copy START -->
 
-        ```bash
-        {container="evaluate-loki-grafana-1"}
-        ```
+      ```bash
+      {container="evaluate-loki-flog-1"}
+      ```
 
-    1. Find all the log lines in the container=flog stream that contain the string "status":
+      <!-- Killercoda copy END -->
 
-        ```bash
-        {container="evaluate-loki-flog-1"} |= `status`
-        ```
+      In Loki, this is a log stream.
 
-    1. Find all the log lines in the container=flog stream where the JSON field "status" is "404":
+      Loki uses [labels](https://grafana.com/docs/loki/<LOKI_VERSION>/get-started/labels/) as metadata to describe log streams.
 
-        ```bash
-        {container="evaluate-loki-flog-1"} | json | status=`404`
-        ```
+      Loki queries always start with a label selector.
+      In the previous query, the label selector is `{container="evaluate-loki-flog-1"}`.
```

Comment:
```suggestion
      In the previous query, the label selector is `container`.
```

Reply:
I think the label selector is the full set of label matchers if the semantics match Prometheus: https://prometheus.io/docs/prometheus/latest/querying/basics/#time-series-selectors

Reply:
Ah, having gone back and looked again, I think you're correct.